src/toast/ops/sss.py

```diff
@@ -97,7 +97,9 @@ def _exec(self, data, detectors=None, **kwargs):
         comm = data.comm.comm_group
 
         for obs in data.obs:
-            dets = obs.select_local_detectors(detectors)
+            dets = obs.select_local_detectors(
+                detectors, flagmask=defaults.det_mask_invalid
```

Comment:
Should we add the detector mask as a separate trait here as well?

Reply:
Done for this and all operators